api/CanvasGradient.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvasgradient",
         "support": {
           "chrome": {
-            "version_added": "6"
+            "version_added": "1"
```

Comment:
I'm hoping this will be confirmed by https://github.com/foolip/mdn-bcd-collector/pull/2334 but I don't have Chrome 1 to test with myself.